package-lock.json

```diff
@@ -5,7 +5,6 @@
     "requires": true,
     "packages": {
         "": {
-            "name": "sonora",
```

Comment:
I don't think this needs to be removed.
I'm using `node` v18 now, which I think is the version that started adding this package name (or maybe it was `npm` v8 🤷 ).

Reply:
Sorry about that, didn't realize my node install was so out of date. I upgraded it and npm and re-ran `npm install` and that line re-appeared. I've pushed it back up.

Reply:
:np: